packages/teuchos/parameterlist/src/Teuchos_ParameterListAcceptorDefaultBase.hpp

```diff
@@ -58,7 +58,7 @@ namespace Teuchos {
  * using the non-virtual protected members <tt>setMyParamList()</tt> and
  * <tt>getMyParamList()</tt>.
  */
-class ParameterListAcceptorDefaultBase : virtual public ParameterListAcceptor {
+class TEUCHOSPARAMETERLIST_LIB_DLL_EXPORT ParameterListAcceptorDefaultBase : virtual public ParameterListAcceptor {
```

Comment:
Okay, we are just adding an export.  I can't see how this can break an existing windows build.